packages/app/src/components/DimensionsPanel/Dialogs/DialogManager.js

```diff
@@ -303,14 +306,50 @@ export class DialogManager extends Component {
                               }),
                       }
                     : dimensionProps
-                content = (
+                const dimensionSelector = (
                     <DataDimension
                         displayNameProp={displayNameProperty}
                         selectedDimensions={selectedItems}
                         infoBoxMessage={infoBoxMessage}
                         {...props}
                     />
                 )
+                const dataTabs = [
+                    ITEM_ATTRIBUTE_VERTICAL,
+                    ITEM_ATTRIBUTE_HORIZONTAL,
+                ].includes(dialogId) ? (
```

Comment:
This is adding both tabs regardless of which item attribute is matched, is that correct?

Reply:
Yeah this is adding both tabs (screenshot) to be displayed and the current tab (depending on which layout item was clicked) is then getting selected by `selected={key === dialogId}` further down.
![image](https://user-images.githubusercontent.com/12590483/101159288-76d48700-362d-11eb-9cde-b31cad76e423.png)

Reply:
👍